src/frontend-scripts/components/section-main/Gamechat.jsx

```diff
@@ -652,10 +652,28 @@ class Gamechat extends React.Component {
 				return null;
 			}
 		};
+		const FollowRemakeButton = () => {
+			if (gameInfo.general.isRemade) {
+				const onClick = () => {
+					window.location.href = `#/table/${gameInfo.general.uid}Remake`;
+					window.location.reload();
```

Comment:
Are you sure you need this?  I'd rather avoid hard reloads.  Router should catch line 658 I think

Reply:
Router does catch it correctly but it doesn't offer to take a seat without that.

Reply:
Cool good enough